sdcflows/fieldmaps.py

```diff
@@ -404,16 +404,20 @@ def __attrs_post_init__(self):
         # Register this estimation method
         if not self.bids_id:
             # If not manually set, try to get it from BIDS metadata
-            bids_ids = set(
-                [
-                    f.metadata.get("B0FieldIdentifier")
-                    for f in self.sources
-                    if f.metadata.get("B0FieldIdentifier")
-                ]
-            )
-            if len(bids_ids) > 1:
+            b0_ids = [
+                listify(f.metadata.get("B0FieldIdentifier"))
+                for f in self.sources
+                if f.metadata.get("B0FieldIdentifier")
+            ]
+            bids_ids = set.intersection(*b0_ids)
```

Comment:
I wanted `b0_ids` to be lists for the sake of reporting, so I changed them from sets.

```suggestion
            bids_ids = set.intersection(*map(set, b0_ids))
```